packages/amplify-codegen/src/utils/validateAmplifyFlutterCoreLibraryDependency.js

```diff
@@ -0,0 +1,9 @@
+const { validateAmplifyFlutterVersion } = require('./validateAmplifyFlutterVersion');
+
+const MINIMUM_VERSION_CONSTRAINT = '>= 0.4.0 || >= 0.4.0-rc.2';
```

Comment:
nit/question: Could the rc version be 0.4.0-rc.1 instead of 2? Unless I'm missing some reason for wanting 2.